vpos/sdk/python/README

```diff
@@ -0,0 +1,163 @@
+
+# Bancard VPOS v1.0 - Bancard Python Connector v0.4 library
+
+## Getting Started
+
+This library allows developers to integrates their Python backend applications to the Bancard VPOS API.
+
+This library works with the following Python versions: 2.6, 2.7, 3.3, 3.4, 3.5, 3.6
+
+### Prerequisites
+
+See the requirements.txt file to see which Python libraries will be required.
+
+### Usage in Staging or Production
+
+You can either include this library from:
+
+```
+https://github.com/vcajes/bancard-connector-python
+```
+
+Or install the library from the [PYPI repository](https://pypi.python.org/pypi/bancardconnectorpython/):
+
+```
+pip3 install bancardconnectorpython
+```
+
+### Usage in development
+
+* Download and install [Python (2.6 <= version <= 3.6)](https://www.python.org/downloads/).
+* Run `pip install bancardconnectorpython`.
+* Import and use library in your source code:
+```
+from decimal import Decimal
+import bancardconnectorpython
+
+# this will autoconfigure the connector from the following OS environment variables:
+# BANCARD_ENVIRONMENT=sandbox|production
+# BANCARD_PUBLIC_KEY=your_public_key
+# BANCARD_PRIVATE_KEY=your_private_key
+bancard_api = bancardconnectorpython.connector()
+
+# or you could just create your own BancardAPI
+bancard_api = bancardconnectorpython.BancardAPI(environment=BancardAPI.ENVIRONMENT_SANDBOX, public_key=your_public_key, private_key=your_private_key)
```

Comment:
After the last change, ENVIRONMENT_SANDBOX was moved to constants.py. Therefore, you should change the assignation to environment=ENVIRONMENT_SANDBOX